mentat/parsers/git_parser.py

```diff
@@ -72,6 +77,9 @@ def parse_string(self, git_diff: str) -> ParsedLLMResponse:
                         end_line = start_line + 1
                     else:
                         end_line = start_line + int(a_b[1])
+                    # TODO: This breaks some of the parser tests, but is required to make
+                    # an insertion diff valid? Need to investigate.
+                    end_line += 1  # FileEdits use exclusive end_line
```

Comment:
@jakethekoenig I had to add this to get my sample_eval's diff (insert 2 lines) to work. I wonder if we adjusted file_edits to use exclusive end_line and this class was missed? Or maybe I need to dig more 😄